tools/rebasehelpers/util/git.go

```diff
@@ -118,6 +118,8 @@ type File string
 func (f File) HasVendoredCodeChanges() bool {
 	return strings.HasPrefix(string(f), "Godeps/_workspace") ||
 		strings.HasPrefix(string(f), "vendor") ||
+		strings.HasPrefix(string(f), "glide.lock") ||
+		strings.HasPrefix(string(f), "glide.yaml") ||
```

Comment:
does this mean that I'd have to put glide.yaml changes into a bump commit?

Reply:
it didn't have that effect when I tested it, it just tolerated it in a bump commit, but now I'm not sure... I'll drop this... the other bits were more important